source/test/F0.Minesweeper.Logic.Tests/LocationShuffler/GuidLocationShufflerTest.cs

```diff
@@ -1,14 +1,21 @@
-using System;
-using System.Collections.Generic;
 using F0.Minesweeper.Logic.Abstractions;
 using F0.Minesweeper.Logic.LocationShuffler;
-using FluentAssertions;
-using Xunit;
 
 namespace F0.Minesweeper.Logic.Tests.LocationShuffler
 {
 	public class GuidLocationShufflerTest
 	{
+		[Fact]
+		public void ThrowIfNull()
+		{
+			ILocationShuffler locationShufflerUnderTest = new GuidLocationShuffler();
+
+			Func<IReadOnlyCollection<Location>> act = () => locationShufflerUnderTest.ShuffleAndTake(null!, 0);
+
+			act.Should().ThrowExactly<ArgumentNullException>()
+				.And.ParamName.Should().Be("allLocations");
```

Comment:
I guess this is the same discussion as [here](https://github.com/Flash0ver/F0.Minesweeper/pull/102#discussion_r775601922) and [here](https://github.com/Flash0ver/F0.Minesweeper/pull/102#discussion_r775602385).

Reply:
It is